Assets/StreamingAssets/Quests/K0C01Y00.txt

```diff
@@ -121,7 +121,7 @@ Message:  1013
  freelancers.
  
  
-<ce>                          -- The Thieves Guild
+<ce>                          The Thieves Guild
 
```

Comment:
Does the "--" cause some kind of problem? You've removed a few of these where a letter is being signed with a dash in front of a name.

Reply:
The dash indicates a comment in TEMPLATE. My parser will exclude a line when a dash is first non-whitespace character. I'm not 100% sure, as I'm not in front of my dev environment right now, but I think the lines that start with <ce> might be OK. But the left-aligned cases with only whitespace in front of them will be excluded.

Reply:
The -- edits I made were done blindly. Since this was my first pass I just went for the things that were most obvious without testing them one-by-one, and I've always been wary of using the -- at the start of a line. I wasn't watching whether a <ce> preceded them TBH.